WorkflowCombine/TestingTests/PublisherTests.swift

```diff
@@ -12,6 +12,7 @@ import WorkflowTesting
 import XCTest
 @testable import WorkflowCombineTesting
 
+@available(iOS 13.0, macOS 10.15, *)
```

Comment:
why were all these availability annotations needed?

Reply:
I added the testing target for those to the package and because the package is iOS 12+ right now.  You can't set minimum OS targets for products only packages unfortunately.